### PR TITLE
Frontend templates, models, forms added

### PR DESCRIPTION
**Pull request by Thek10patil**
Best code ever

### CODE REVIEW

**Review by vibnimit — DISMISSED** [review]
Nice

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (AdityaGovardhan) [inline .idea/.gitignore]: .idea is something specific to your IDE. Avoid creating a new .gitignore, add this folder and workspace.xml to .gitignore

File: .idea/image-search-engine.iml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AdityaGovardhan) [inline .idea/image-search-engine.iml]: Add .idea to .gitignore in root folder

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (AdityaGovardhan) [inline .idea/inspectionProfiles/profiles_settings.xml]: Add .idea to .gitignore in root folder

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AdityaGovardhan) [inline .idea/modules.xml]: Add .idea to .gitignore in root folder

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AdityaGovardhan) [inline .idea/vcs.xml]: Add .idea to .gitignore in root folder

**Review by AdityaGovardhan — COMMENTED** [review]
1) Remove conda_start.sh if not required
2) Check the requirement of db.sqlite3, if not required, remove
3) @vibnimit mentioned in call, will keeping folder name as views create a problem?

File: .idea/vcs.xml (hunk 5 above)
Comment (Thek10patil) [reply .idea/vcs.xml]: done

File: .idea/modules.xml (hunk 4 above)
Comment (Thek10patil) [reply .idea/modules.xml]: done

File: .idea/inspectionProfiles/profiles_settings.xml (hunk 3 above)
Comment (Thek10patil) [reply .idea/inspectionProfiles/profiles_settings.xml]: done

File: .idea/image-search-engine.iml (hunk 2 above)
Comment (Thek10patil) [reply .idea/image-search-engine.iml]: done

File: .idea/.gitignore (hunk 1 above)
Comment (Thek10patil) [reply .idea/.gitignore]: done

File: src/tasks/__init__.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+from .task1 import *
+from .task2 import *
+from .task3 import *
+from .task4 import *
+from .task5 import *
+from .task6 import *
```
Comment (vibnimit) [inline src/tasks/__init__.py:6]: Try to avoid using import * it leads to circular import dependency errors

**Review by vibnimit — APPROVED** [review]

**Review by AdityaGovardhan — APPROVED** [review]
gtg

**Pull request merged**